From bf2b9410392bdb4427e25e60cbd45a29b47d8f9b Mon Sep 17 00:00:00 2001
From: Rohit Athavale <rohit.athavale@xilinx.com>
Date: Tue, 27 Feb 2018 15:34:50 -0800
Subject: [PATCH 0292/1852] media-bus: uapi: Add YCrCb 420 media bus format

commit 09aa09890ceccf72659824728e7982df06ef30af from
https://github.com/Xilinx/linux-xlnx.git

This commit adds a YUV 420 media bus format. Currently, one
doesn't exist. VYYUYY8_1X24 does not describe the way the pixels are
sent over the bus, but is an approximation.

Signed-off-by: Rohit Athavale <rohit.athavale@xilinx.com>
Signed-off-by: Satish Kumar Nagireddy <satishna@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: waiting (media)
Link: https://www.spinics.net/lists/linux-media/msg133380.html
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/uapi/linux/media-bus-format.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/media-bus-format.h b/include/uapi/linux/media-bus-format.h
index 2a6b253cfb05..7c8f76436fed 100644
--- a/include/uapi/linux/media-bus-format.h
+++ b/include/uapi/linux/media-bus-format.h
@@ -108,6 +108,7 @@
 #define MEDIA_BUS_FMT_YUV12_1X36		0x2029
 #define MEDIA_BUS_FMT_YUV16_1X48		0x202a
 #define MEDIA_BUS_FMT_UYYVYY16_0_5X48		0x202b
+#define MEDIA_BUS_FMT_VYYUYY8_1X24		0x202c
 
 /* Bayer - next is	0x3021 */
 #define MEDIA_BUS_FMT_SBGGR8_1X8		0x3001
-- 
2.31.1

